monitoring/bin/deploy_monitoring_viya.sh

```diff
@@ -89,5 +89,5 @@ else
   log_warn "Prometheus Operator not found. Skipping deployment of ServiceMonitors."
 fi
 
-deployV4MInfo "$VIYA_NS" "v4m-viya"
+deployV4MInfo "$VIYA_NS" "v4m-monitoring-$VIYA_NS"
```

Comment:
I'm wondering if we really want to lose the  string "viya" here.  Without it, we can only infer that that this Helm release is a tenant-level one because there is only one string after the "v4m-".  But what if a namespace name includes a hyphen (e.g. kube-system)?  Is the Helm release v4m-kube-system a namespace-level release (for a namespace called 'kube-system') or is it a tenant-level release (for a tenant 'system' within the 'kube' namespace)?  

When constructing the ES index names, we used underscores and plus signs to separate namespace names and tenant names, since neither of those characters was acceptable within either namespace or tenant names.  I'm not pushing to duplicate that here necessarily, just raising the general issue.  Are there limitations on which characters can be included in a Helm release name?

On the other hand, Helm releases are namespace-scoped, so maybe that helps to some degree.  Maybe until someone deploys Viya into a namespace called "logging" (or "monitoring"). 😉